### ISSUE TITLE
test_pl_validation_translation fails without Internet access (2.4.0)

### ISSUE THREAD

**Issue by mgorny** [sissaschool/xmlschema#359]
When testing 2.4.0, I'm getting the following new test failure:

```pytb
======================================================================
FAIL: test_pl_validation_translation (test_translations.TestTranslations)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/portage/dev-python/xmlschema-2.4.0/work/xmlschema-2.4.0/tests/test_translations.py", line 128, in test_pl_validation_trans
lation
    assert errors == expected_errors
AssertionError

----------------------------------------------------------------------
```

Scrolling back, I see the following errors:

```pytb
test_pl_validation_translation (test_translations.TestTranslations) ... /tmp/portage/dev-python/xmlschema-2.4.0/work/xmlschema-2.4.0/tests/test_translations.py:122: XMLSchemaImportWarning: Import of namespace 'http://crd.gov.pl/xml/schematy/dziedzinowe/mf/2020/07/06/eD/DefinicjeTypy/' from ['http://crd.gov.pl/xml/schematy/dziedzinowe/mf/2020/07/06/eD/DefinicjeTypy/StrukturyDanych_v7-0E.xsd'] failed: cannot access to resource 'http://crd.gov.pl/xml/schematy/dziedzinowe/mf/2020/07/06/eD/DefinicjeTypy/StrukturyDanych_v7-0E.xsd': [Errno -3] Temporary failure in name resolution.
  schema = XMLSchema(
/tmp/portage/dev-python/xmlschema-2.4.0/work/xmlschema-2.4.0/tests/test_translations.py:122: XMLSchemaImportWarning: Import of namespac
e 'http://crd.gov.pl/xml/schematy/struktura/2009/11/16/' from ['http://crd.gov.pl/xml/schematy/struktura/2009/11/16/struktura.xsd'] failed: cannot access to resource 'http://crd.gov.pl/xml/schematy/struktura/2009/11/16/struktura.xsd': [Errno -3] Temporary failure in name resolution.
  schema = XMLSchema(
/tmp/portage/dev-python/xmlschema-2.4.0/work/xmlschema-2.4.0/tests/test_translations.py:122: XMLSchemaImportWarning: Import of namespace 'http://crd.gov.pl/xml/schematy/dziedzinowe/mf/2021/04/14/eTW/Commons/' from ['http://crd.gov.pl/xml/schematy/dziedzinowe/mf/2021/04/14/eTW/Commons/schemat_commons.xsd'] failed: cannot access to resource 'http://crd.gov.pl/xml/schematy/dziedzinowe/mf/2021/04/14/eTW/Commons/schemat_commons.xsd': [Errno -3] Temporary failure in name resolution.
  schema = XMLSchema(
FAIL
```

FWICS other tests requiring Internet are skipped cleanly, so I suppose there's a skip missing here.

**Comment by brunato** [comment]
Yes, the test will be skipped in the next release:

```python
    @unittest.skipIf(SKIP_REMOTE_TESTS, "Remote networks are not accessible")
    def test_pl_validation_translation(self):
```

thank you

**Comment by mgorny** [comment]
Thank you!

**Comment by brunato** [comment]
Fixed by v2.5.0.

Also fixed other problems in the same test (rigid filepaths and feasible remote import failure that can happens if the addressed site doesn't respond).
